.storybook/Popover.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import {
+  Popover,
+  Button,
+  IconButton,
+  CloseButton,
+  InputField,
+  Label,
+  Input,
+  Card,
+  Flex,
+  Box,
+  Heading,
+  Text,
+  Hide
+} from '../src'
+
+const withState = C =>
+  class extends React.Component {
+    constructor(props) {
+      super()
```

Comment:
It's usually not necessary, but it could be good for consistency in code style – see https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e